src/utils/compactall.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from lib.ursadb import UrsaDb
+import time
+import argparse
+
+
+def main() -> None:
+    logging.basicConfig(level=logging.INFO)
+
+    parser = argparse.ArgumentParser(description="Keep the database lean.")
```

Comment:
Worth considering if these shouldn't be just UrsaDB feature/more advanced command

Reply:
There is a benefit ofdoing this in another script: this script is NON-transactional. Ursadb tries hard to make every operation atomic. It's usually a nice property, but when compacting of a large database takes days, you don't want to lose all progress because of server reboot, for example.

See also the second script (reindex_local) that also has this non-property.

Of course, we could also make core ursadb non-transactional (it's very tempting), but so far I've resisted this temptation!